openlibrary/catalog/add_book/load_book.py

```diff
@@ -283,7 +299,18 @@ def author_import_record_to_author(
     :return: Open Library style Author representation, either existing Author with "key",
              or new candidate dict without "key".
     """
-    assert isinstance(author_import_record, dict)
+    try:
```

Comment:
We checked, and the one place this method is used, it ensure the input is a `dict`, so it's cleaner to rely on the typehint here since that appears to be actually enforced!